tests/Traits/MocksContract.php

```diff
@@ -96,6 +96,7 @@ protected function getMockedShipment(
         if ($service) {
             $shipment->method('getService')->willReturn($service);
         }
+        $shipment->method('calculateVolumeInMm3')->willReturn($volumeInMm3);
```

Comment:
Lets also unit test the `calculateVolumeInMm3` and `calculateVolumeInDm` methods in a unit test?